messages/en.json

```diff
@@ -2,106 +2,202 @@
     "no_victims": [
         "The body of a young penguin pet is found.",
         "Paw prints are found circling around the village.",
+        "Some paw prints are found leading to the village square.",
         "A pool of blood and wolf paw prints are found.",
         "The body of a slain cat is found.",
         "Some house doors have been opened.",
+        "A few houses have fresh blood on them.",
         "A scent much like that of a wolf permeates the air.",
         "Half-buried wolf droppings are found.",
         "Traces of wolf fur are found.",
+        "It seems a wolf took a nap here.",
+        "The village seems eerily quiet.",
```

Comment:
Please use the passive tense (all the other messages use it). The way it's said makes it seem like a narrator says it, instead of people constating it.